examples/create_libraries/test/hw_mp_browser_test.dart

```diff
@@ -0,0 +1,14 @@
+// To run this test:
+// cd create_libraries
+// pub run test -p chrome
+@Tags(['browser'])
```

Comment:
Why tag it `'browser'`? asking because I haven't done that in my code, does it mean anything?

Reply:
It's required on site-www to get browser tests detected and run.